packages/editor/src/components/collab-sidebar/comment-form.js

```diff
@@ -36,6 +36,7 @@ function CommentForm( { onSubmit, onCancel, thread, submitButtonText } ) {
 				__nextHasNoMarginBottom
 				value={ inputComment ?? '' }
 				onChange={ setInputComment }
+				aria-label={ __( 'Comment' ) }
```

Comment:
```suggestion
				label={ __( 'Comment' ) }
				hideLabelFromVision
```

Let's use the official prop to visually hide the label.